aioaquarea/data.py

```diff
@@ -506,3 +506,11 @@ async def set_temperature(
         :param temperature: The temperature to set
         :param zone_id: The zone id to set the temperature for
         """
+
+    @abstractmethod
+    async def set_quiet_mode(
+        self, mode: int
```

Comment:
Could you define mode as an Enum `QuietMode`, similar to the way `OperationStatus' or 'OperationMode' are defined with all the posible values?https://github.com/cjaliaga/aioaquarea/blob/617037187915c88e8e7b6ca5ce5d120c85d3d10f/aioaquarea/data.py#L32-L42

Reply:
I added the enum, thanks!